src/pages/guides/using/marketo/marketo-data-streams.md

```diff
@@ -0,0 +1,78 @@
+# Subscribing to the Marketo Events with Adobe I/O Events
+
+These instructions describe how to set up and get started using Adobe I/O Events for the Marketo Data Streams.
+
+- [Introduction](#introduction)
+- [Subscribing to Marketo Events](#subscribing-to-marketo-events)
+- [Developer Guidelines](#developer-guidelines)
+- [Debug](#debug)
+
+## Introduction
+
+In order to allow for additional scalability and further programmatic automation of the Marketo platform, we have introduced the Marketo Data Streams.  The data streams are available to all CET and Performance Plus customers.  There are currently five data streams:
+
+- Lead Activity Data Stream
+- Metrics Data Stream (Alpha)
+- Notification Data Stream (Beta)
+- Observability Data Stream (Beta)
+- User Audit Data Stream (Beta)
+
+At this time, only the Notification, Observability, and User Audit streams are available within Adobe I/O Events, the Lead and Metrics streams can be subscribed to via alternate paths.
+
+Note: The Marketo Data Streams are currently in Beta
+
+## Subscribing to Marketo Events
+
+See the [Adobe I/O Events Docs](/src/pages/index.md)
+
+Here are some basic getting started instructions that will be the same for each of the available Marketo data streams in I/O Events.
+
```

Comment:
Not required since it will be in the tree on the left but you make a good point, it would probably make sense to have links to our audit and observability stream pages in our top level page.